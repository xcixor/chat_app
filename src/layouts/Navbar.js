import React from "react";
import PropTypes from "prop-types";
import { Nav, Container } from "react-bootstrap";

export default function Navbar({ currentUser }) {
	return (
		<Container className="py-2">
			<Nav className="justify-content-between align-items-center">
				{currentUser ? (
					<Nav.Item>
						<Nav.Link href="/" className="p-0">
							Hi {currentUser}
						</Nav.Link>
					</Nav.Item>
				) : (
					""
				)}
				<Nav.Item>
					<Nav.Link
						href="/history"
					>
						History
					</Nav.Link>
				</Nav.Item>
				<Nav.Item>
					<Nav.Link
						href="/"
						target="_blank"
						className="p-2 btn btn-primary text-white"
					>
						Connect as New User
					</Nav.Link>
				</Nav.Item>
			</Nav>
		</Container>
	);
}

Navbar.propTypes = {
	currentUser: PropTypes.string,
};
