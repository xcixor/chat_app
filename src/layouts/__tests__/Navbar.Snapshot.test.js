import React from "react";
import Navbar from "../Navbar";
import renderer from "react-test-renderer";

it("Renders correctly", () => {
	const tree = renderer.create(<Navbar />);

	expect(tree).toMatchSnapshot();
});
