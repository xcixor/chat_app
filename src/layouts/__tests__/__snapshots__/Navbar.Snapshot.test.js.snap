// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders correctly 1`] = `
<div
  className="py-2 container"
>
  <div
    className="justify-content-between align-items-center nav"
    onKeyDown={[Function]}
  >
    <div
      className="nav-item"
    >
      <a
        className="p-2 btn btn-primary text-white nav-link"
        data-rr-ui-event-key="/"
        disabled={false}
        href="/"
        onClick={[Function]}
        target="_blank"
      >
        Connect as New User
      </a>
    </div>
  </div>
</div>
`;
